ci/deps/azure-macos-36.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   # pandas dependencies
   - beautifulsoup4
   - bottleneck
+  - cftime  # Needed for downstream xarray.CFTimeIndex test
```

Comment:
these should not be in every CI yaml 
rather u skip the test if if it’s not installed

Reply:
Yes, I understand; I was just following [@jbrockmendel's recommendation](https://github.com/pandas-dev/pandas/pull/32905#discussion_r396129937).  Which CI yaml should I put this in?

Reply:
remove from all
except the environment.yaml